src/include/migraphx/op/unary.hpp

```diff
@@ -13,7 +13,15 @@ struct unary : op_name<Derived>
     shape compute_shape(std::vector<shape> inputs) const
     {
         check_shapes{inputs}.has(1);
-        return inputs.at(0);
```

Comment:
This shouldn't change. It should only change for `unary_device`.

Reply:
If only changing the unary_device, then CPU and GPU implementations will generate different output shape in some cases.